src/App.js

```diff
@@ -46,6 +48,13 @@ function App() {
 
   function addToShoppingCard(card) {
     const objectExists = shoppingCard.find((object) => object.id === card.id);
+    const current = new Date();
+    const date = `${current.getDate()}/${
+      current.getMonth() + 1
+    }/${current.getFullYear()}`;
+
+    const today = new Date();
```

Comment:
You already have a date object in line 51 called `current`. You don't need an additional one here.

Reply:
done